pkg/util/util.go

```diff
@@ -98,3 +128,96 @@ func AddAnnotation(item runtime.Unstructured, annotation, value string) {
 
 	metadata.SetAnnotations(annotations)
 }
+
+func IsVMIPaused(vmi *kvv1.VirtualMachineInstance) bool {
+	for _, c := range vmi.Status.Conditions {
+		if c.Type == kvv1.VirtualMachineInstancePaused {
+			return true
```

Comment:
Don't you need to check the status? what if it is
```
 type:Paused, Status: false
 ```

Reply:
You're right, the status should also be checked. Fixing.